harness/determined/keras/_tf_keras_context.py

```diff
@@ -40,6 +49,61 @@ def __init__(self, env: det.EnvContext, hvd_config: horovod.HorovodContext):
         self._wrapped_optimizers = []  # type: List[tf.keras.optimizers.Optimizer]
         self._compiled_optimizer = None  # type: Optional[tf.keras.optimizers.Optimizer]
 
+        # The following attributes may be configured via configure_fit().  Defaults match the
+        # normal keras.fit() defaults.
+        self._fit_verbose = True
+        self._fit_class_weight = None
+        self._fit_workers = 1
+        self._fit_use_multiprocessing = False
+        self._fit_max_queue_size = 10
+
+    def configure_fit(
+        self,
+        verbose: Optional[bool] = None,
+        class_weight: Any = _arg_not_provided,
+        workers: Optional[int] = None,
+        use_multiprocessing: Optional[bool] = None,
+        max_queue_size: Optional[bool] = None,
+    ) -> None:
+        """
+        Configure the following arguments to ``model.fit()``:
+
+           -  ``verbose``: defaults to ``True``. The Determined implementation of ``verbose=True``
+              will output periodic text updates rather than the usual ``tqdm`` status bar (which
+              does not render well in trial logs).
+
+           -  ``class_weight``: defaults to ``None``
+
+           -  ``workers``: defaults to 1. Ignored for ``tf.data.dataset`` data.
+
+           -  ``use_multiprocessing``: defaults to `False`. Ignored for ``tf.data.dataset`` data.
+
+           -  ``max_queue_size``: defualts to `10`. Ignored for ``tf.data.dataset`` data.
+
+        Note that any keyword arguments which are not provided will not overwrite any previously
```

Comment:
non-blocking: probably useful to explain this matters only if calling `configure_fit()` multiple times.

Reply:
done